code/modules/events/disease_outbreak.dm

```diff
@@ -1,41 +1,75 @@
+GLOBAL_VAR(current_pending_disease)
 /datum/event/disease_outbreak
-	announceWhen = 180
 	/// The type of disease that patient zero will be infected with.
 	var/datum/disease/D
```

Comment:
I know it was in place before you reworked it, but could you change this variable name from `D` to something more sensible, like `disease`? A single letter instance variable is icky.